mm/iob/iob_alloc.c

```diff
@@ -93,7 +93,7 @@ static FAR struct iob_s *iob_alloc_committed(enum iob_user_e consumerid)
  *
  ****************************************************************************/
 
-static FAR struct iob_s *iob_allocwait(bool throttled,
+static FAR struct iob_s *iob_allocwait(bool throttled, unsigned int timeout,
```

Comment:
should me somehow make a hint that `timeout` has ms resolution?

Reply:
OK, I will add describe of timeout parameter in code comment.